mailing-list-api/server.js

```diff
@@ -0,0 +1,77 @@
+process.env.PORT = process.env.PORT || 9090;
+const express = require("express");
+const cors = require("cors");
+
+const server = express();
+
+server.use(express.json()); // needed to parse JSON data
+server.use(cors()); // middleware for security - makes Express.js backend accessible to frontend applications running on different domains, while still following CORS security rules
+
+const mailingListObject = require("./mailing-lists");
+
+server.get("/", function (request, response) {
+  response.send("Mailing list server");
+});
+
+server.get("/lists", function (request, response) {
+  let mailingListKeys = Object.keys(mailingListObject);
+  if (mailingListKeys) {
+    response.status(200).send(mailingListKeys);
+    // response.status(200).send(JSON.stringify(mailingList)); // HELP!! Instructions say response should be "a JSON body of all the existing list names". Do I need to stringify?
```

Comment:
Take a look at the `json()` function provided by express

https://expressjs.com/en/4x/api.html#res.json